api/app/signals/apps/api/v1/urls.py

```diff
@@ -97,5 +99,9 @@
 
         # Search
         re_path('search/?$', SearchView.as_view({'get': 'list'}), name='elastic-search'),
+
+        # csv
+        re_path('csv/?$', PrivateCsvViewSet.as_view({'get': 'download'}), name='download-csv'),
```

Comment:
Is this not redundant given line 55 and the fact that `/download` is marked as an `@action` in the viewset?

Reply:
check removed 55, since 104 is required

Reply:
I removed line 55, only want to allow download